### ISSUE TITLE
🚀 [Feature] Backend: Generate invite links for an user

### ISSUE THREAD

**Issue by ulfgebhardt** [dreammall-earth/dreammall.earth#2350]
## 🚀 Feature

Backend: Generate invite links for an user

We need to be able to generate invitation links which are bound to an user.

It seems smart to use an cryptic hash for this purpose. We should reuse `nanoid` already in the project

Build the following queries:
- Generate (personal) Invite link as user (one link per user is enough for now)
- ~~Delete (own) Invite link as user~~
- ~~Get list of generated invite links~~
- Validate invite code as anyone to receive validity and inviting user information

We do not need to implement any limitations for now.
- ~~If a link is usable only once can be specified in redeem at a later point.~~
- ~~The number of link an user can generate can be limited alter on.~~

**Comment by ulfgebhardt** [comment]
closing in favor of https://github.com/dreammall-earth/dreammall.earth/issues/2255
